BALSAMIC/containers/vcf2cytosure/vcf2cytosure.yaml

```
@@ -1 +1 @@
-- vcf2cytosure=0.8.1
+- vcf2cytosure=0.8
```

The [version is 0.8.1 ](https://github.com/NBISweden/vcf2cytosure/blob/master/__version__.py)actually, the only thing it has been not bumped... 
```suggestion
- vcf2cytosure=0.8.1
```